src/services/account/useAccountDetails.ts

```diff
@@ -53,23 +73,7 @@ export const PaymentMethodSchema = z
       expYear: z.number(),
       last4: z.string(),
     }),
-    billingDetails: z
-      .object({
-        address: z
-          .object({
-            city: z.string().nullable(),
-            country: z.string().nullable(),
-            line1: z.string().nullable(),
-            line2: z.string().nullable(),
-            postalCode: z.string().nullable(),
-            state: z.string().nullable(),
-          })
-          .nullable(),
-        email: z.string().nullable(),
-        name: z.string().nullable(),
-        phone: z.string().nullable(),
-      })
-      .nullish(),
+    billingDetails: BillingDetailsSchema.nullish(),
```

Comment:
If these values are coming from GQL they shouldn't be able to be `undefined`, so we probably only need `nullable` here

Reply:
Sure, works for me

Reply:
Updated in https://github.com/codecov/gazebo/pull/2955/commits/82e9f90d633286cf81a07be6059ca592daaaef75